common/app/routes/challenges/components/TranslationAlert.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { PropTypes } from 'react';
+import PureComponent from 'react-pure-render/component';
+import { Alert, Grid, Row, Col } from 'react-bootstrap';
+
+import supportedLanguages from '../../../../utils/supported-languages';
+
+const propTypes = {
+  lang: PropTypes.string.isRequired
+};
+
+export default class TranslationAlert extends PureComponent {
+  render() {
+    const { lang } = this.props;
+    const langString = supportedLanguages[lang];
+    const href = 'https://github.com/FreeCodeCamp/TranslationExpansion/issues';
```

Comment:
I would recommend we link to our forum article instead, many challenges already have open issues. This might lead to people opeing duplicate issues instead of working on the open ones.

The below is a better option to get them started with the translation efforts IMHO.
http://forum.freecodecamp.com/t/guidelines-for-translating-free-code-camp-to-any-language/19111